src/dex/uniswap-v2/uniswap-v2.ts

```diff
@@ -271,13 +274,19 @@ export class UniswapV2
       callDecoder,
       this.decoderIface,
     );
-    pair.pool.addressesSubscribed.push(pair.exchange!);
+    this.pairToPoolMap[this.getKeyForPair(token0, token1)].addressesSubscribed.push(pair.exchange!);
```

Comment:
Why not save `this.getKeyForPair(token0, token1)` to a variable?

Reply:
@skypper updated